src/tree.ts

```diff
@@ -305,12 +305,13 @@ export class ProjectOutlineProvider implements vscode.TreeDataProvider<BaseNode>
        if (!pathInSettings) {
          return `${kind}: [Unset]`;
        }
-
-       const pathBase: string | undefined = (searchInPath && path.parse(pathInSettings).dir === "") ? path.parse(pathInSettings).base : undefined;
+       
+       const pathInSettingsToTest = process.platform === "win32" && !pathInSettings.endsWith(".exe") && kind === "Make" ? pathInSettings.concat(".exe") : pathInSettings;
+       const pathBase: string | undefined = (searchInPath && path.parse(pathInSettingsToTest).dir === "") ? path.parse(pathInSettingsToTest).base : undefined;
        const pathInEnv: string | undefined = pathBase ? (path.join(util.toolPathInEnv(pathBase) || "", pathBase)) : undefined;

```

Comment:
Minor: I know I missed some linter setting somewhere. VSCode squiggles for me that it wants all the types for variables so that's why we define all the types when we declare. Let's keep pathInSettingsToTest consistent (or until we decide we don't want to do this and set VSCode to not complain about that).